tools/release/changelog-root-template.hbs

```diff
@@ -0,0 +1,15 @@
+{{> header}}
+
+{{#each packageGroups}}
+### {{title}}
+
+|            |                       |
+| ---------- | --------------------- |
+{{#each commits}}
+| <img src="{{typeImageUrl}}" alt="{{typeDescription}}"> | {{> commit root=@root}} |
+{{/each}}
```

Comment:
I had that idea as well and tried that initially but it didn't look so well. Just my personal opinion. Happy to go with a separate column/or show you an image of how it would look.